pkg/cmd/attestation/logging/logger.go

```diff
@@ -0,0 +1,92 @@
+package logging
```

Comment:
With this coming into `gh`, I'm not sure how much of `logging` package is relevant given that commands are provided the factory, which includes IO and other concerns.

Reply:
Finishing first pass of review, I see the attestation commands have a verbose form and quiet form of output.  Aside from a minor nit regarding "logging" versus IO, I'm wondering how this ties into the native `GH_DEBUG` use

https://github.com/cli/cli/blob/4601d7ff5e8f87be07b7b97aafedc93deeeca3ef/utils/utils.go#L10-L29

Reply:
So I removed the `quiet` and `verbose` flags and updated the logger struct to be called `io.Handler`. The handler will call `IsDebugEnabled()` to set verbosity based on that.